src/css/controls/flags/audioplayer.less

```diff
@@ -16,6 +16,10 @@
         min-height: @mobile-touch-target;
     }
 
+    .jw-spacer {
+        display: none;
+    }
```

Comment:
Why would you do this? Do you want all the controls to be centered in the player?

<img width="1259" alt="screen shot 2019-02-27 at 3 34 24 pm" src="https://user-images.githubusercontent.com/333258/53521061-31bd1180-3aa5-11e9-9af3-5a922ea38855.png">